native/src/components/PoiFiltersModal.tsx

```diff
@@ -78,6 +78,7 @@ type PoiFiltersModalProps = {
   setSelectedPoiCategory: (poiCategory: PoiCategoryModel | null) => void
   currentlyOpenFilter: boolean
   setCurrentlyOpenFilter: (currentlyOpen: boolean) => void
+  poisLength: number
```

Comment:
Optional: I think something like `poisCount` or `numberOfPois` but be a little more clear